src/components/Home.tsx

```diff
@@ -21,11 +25,11 @@ function Home(): JSX.Element {
           /*         showPrev={true|false}                                 */
         >
           {/* Each child element of the Carousel is considered as one "slide", like so */}
-          <div><h1>First</h1> <p>Lorem ipsum</p></div>
+          <div><h1>First</h1></div>
           <div><h1>Second</h1></div>
           <div><h1>Third</h1></div>
-          <div><h1>Fourth</h1></div>
-        </Carousel>
+          <div><h1><button onClick={ () => toggleCarousel(false) } >Final Screen</button> </h1></div>
+        </Carousel> : <YouTube chosenVideo= { chosenVideoPath } setVideo= { chooseVideo } showNext = { toggleCarousel } /> }
```

Comment:
nit (not important, feel free to dismiss), i think its easier to read if its cleanly divided on a new line!
```suggestion
        </Carousel> : 
        <YouTube chosenVideo= { chosenVideoPath } setVideo= { chooseVideo } showNext = { toggleCarousel } /> }
```